recipes-ot3-tools/bmap-tools/bmap-tools_git.bb

```diff
@@ -0,0 +1,47 @@
+# Recipe created by recipetool
+# This is the basis of a recipe and may need further editing in order to be fully functional.
+# (Feel free to remove these comments when editing.)
+
+SUMMARY = "Tools to generate block map (AKA bmap) and flash images using bmap"
+# WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
+# your responsibility to verify that the values are complete and correct.
+#
+# The following license files were not able to be identified and are
+# represented as "Unknown" below, you will need to check them yourself:
+#   debian/copyright
+# NOTE: Original package / source metadata indicates license is: GPLv2
+#
+# NOTE: multiple licenses have been detected; they have been separated with &
+# in the LICENSE value for now since it is a reasonable assumption that all
+# of the licenses apply. If instead there is a choice between the multiple
+# licenses then you should change the value to separate the licenses with |
+# instead of &. If there is any doubt, check the accompanying documentation
+# to determine which situation is applicable.
+LICENSE = "GPLv2 & Unknown"
```

Comment:
Let's get these sorted from the source please. The repo identifies itself on github and in COPYING as gpl-2 only.

Reply:
ok, i guess I can leave both the license files in, but not call the other one unkown